tools/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:18.04
```

Comment:
I may suggest using a more recent version, if possible. `22.04` is now the most recent LTS version.

Reply:
Yeah, normally I would agree to this but the build pipeline also relies on 18.04

https://github.com/supercollider/supercollider/blob/647d7bb23d6211e3d681a29b2f196924d403e290/.github/workflows/actions.yml#L15-L18

and 20.04 is the last version to have clang-format-8 as a binary-package (https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=clang-format-8&searchon=names), otherwise one has to build the version oneself which takes around 10 minutes to build it within the container and by using this version we can skip this energy consuming task. Also 18.04 still receives security updates up to 2028: https://ubuntu.com/about/release-cycle